src/components/Auth/Login.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import "./auth.css";
+const Login = () => {
+  return (
+    <div>
+      <section>
+        <div className='auth-wrapper'>
+          <div className='auth-card-wrapper'>
+            <div className='auth-card'>
+              <div className='heading-auth'>
+                <h1>Login in to Winter</h1>
+              </div>
+              <hr className='auth-card-hr' />
+              <div>
+                <form action='' className='auth-content'>
+                  <label for='email'>
+                    <h3>Enter Your Email</h3>
+                  </label>
+                  <input
+                    placeholder='Enter Your Email'
+                    type='email'
+                    id='email'
+                  />
+                  <label for='pwd'>
+                    <h3>Enter Your Password</h3>
+                  </label>
+                  <input
+                    placeholder='Enter Your Password'
+                    type='password'
+                    id='pwd'
+                  />
+                  <a href=''>
+                    {" "}
```

Comment:
You could avoid adding this `{" "}`. I know it's some bug of prettier but that doesn't look good in code.